packages/cozy-stack-client/src/KonnectorCollection.js

```diff
@@ -3,8 +3,12 @@ import AppCollection from './AppCollection'
 export const KONNECTORS_DOCTYPE = 'io.cozy.konnectors'
 
 class KonnectorCollection extends AppCollection {
-  doctype = KONNECTORS_DOCTYPE
   endpoint = '/konnectors/'
```

Comment:
Nit: you can also put this in the constructor, so there is not a mix between class properties and initialization in the constructor.